### ISSUE TITLE
Enforce explicit boolean property with `react/jsx-boolean-value`

### ISSUE THREAD

**Issue by marlomgirardi** [feedzai/eslint-config-feedzai#26]
WDYT about this rule?

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md

As talked with @lfac-pt, at Pulse the common usage is `<Comp prop={true} />`.

**Comment by JoaoTMDias** [comment]
IMO, I think the true prop should be enforced - since it is easier to understand.
And I also think the same prop with a false value should be omitted if there's no need to pass that (meaning it's the default value).

On the other hand, having only the prop name without true/false values can come as simpler to read.

**Comment by nancruz** [comment]
IMO I think we should always enforce the boolean property. It is more explicit.

**Comment by lfac-pt** [comment]
@marlomgirardi do you want to do a pr with this then?

**Comment by AndreTecFil1991** [comment]
I also agree with enforcing boolean values. It is shorter to their values omitted but sometimes is not clear without having it explicit
